### DISCUSSION TITLE
Add additional font formatting support in Snackbar

### DISCUSSION THREAD

**Discussion by SFSteve** [CommunityToolkit/Maui#1019] [New Feature Discussions]
The current iteration of the Snackbar only allows control of the font size. It would be useful to be able to set the font family and weight for consistency with the rest of the application styling.

**Comment by VladislavAntonyuk** [comment]
you can use any font. Use Font property of SnackbarOptions https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/alerts/snackbar?tabs=android#snackbaroptions

**Reply by SFSteve** [reply]
The Family attribute of SnackbarOptions.Font appears to be read only, am I doing something wrong?

**Reply by VladislavAntonyuk** [reply]
You need to create a new instance of font

**Reply by maslewandowski** [reply]
I tried
`snackbarOptions.Font = Microsoft.Maui.Font.OfSize("text", 14);`

And registered the custom font in MauiProgram.cs like:
`fonts.AddFont("helveticaneueltpro_tt.otf", "text");`

And added the font with build action 'MauiFont' with the name 'helveticaneueltpro_tt.otf'

And it works.

**Reply by VladislavAntonyuk** [reply]
It works starting from versoin 5.0.0

**Reply by SFSteve** [reply]
Working for me now as well, turns out I was still on version 3.0.0. Thanks for the help.
